osu.Game/Rulesets/Scoring/HealthProcessor.cs

```diff
@@ -26,6 +31,12 @@ public abstract class HealthProcessor : JudgementProcessor
         /// </summary>
         public readonly BindableDouble Health = new BindableDouble(1) { MinValue = 0, MaxValue = 1 };
 
+        /// <summary>
+        /// The treshold under which the current health value should be considered low.
+        /// Used by HUD elements to display visual indication that health value is low.
+        /// </summary>
+        public virtual double LowHealthTreshold => 0.20f;
```

Comment:
I see little reason for this to exist here to then be propagated to all score displays (that might not even need it) instead of it being locally in `LowHealthLayer`. This seems like preemptive design. It can be extracted when there's a use-case for it.

Reply:
Since health handling is not the same accross all rulesets, I had implemented this property to set the threshold at which the health should be considered low (the point at which the `LowHealthLayer` should start fade in)
or to completely disable the fading of the component if the ruleset's health system works instead like in taiko. Also this could be used by the `StandardHealthDisplay` to change the bar accent colour when health gets under this threshold.

Reply:
Yeah I'm not sure I like this direction. If the low health layer is going to have ruleset-specific logic then that logic should reside in the rulesets themselves and not in the now-as-reusable-as-possible health processor (the goal with them was to unify health as much as possible, aside from taiko which is so different to begin with that it's unviable).

I'd even go as far as to say say that the proper place for the layers is in the ruleset-specific playfields (if one doesn't want the layer, it just doesn't add it, but if another one does, it "automagically" somehow gets the health processor and knows what its alpha needs to be based on that).